app/services/ssm_parameters.rb

```diff
@@ -18,6 +18,13 @@ def delete_parameter
                              })
   end
 
+  def get_invalid_parameters(ssm_paths)
+    response = client.get_parameters({
+                                       names: ssm_paths
```

Comment:
@Yuuki77 Sorry it took me so long to re-review this. I found another small issue, if the heritage has no SSM parameters this array will be empty. This causes AWS SDK to throw an error, and results in Barcelona returning a 500 error.

```
F, [2021-07-29T04:08:49.816365 #1] FATAL -- : [7af8f737-d13b-4a03-8044-5bb3fbd38a85] Aws::SSM::Errors::ValidationException (1 validation error detected: Value '[]' at 'names' failed to satisfy constraint: Member must have length greater than or equal to 1.):
```

You can find this error occuring here:  https://ap-northeast-1.console.aws.amazon.com/cloudwatch/home?region=ap-northeast-1#logsV2:log-groups/log-group/Barcelona$252Fdefault$252Fbarcelona/log-events/barcelona$252Fbarcelona-web$252F4fc663b8b6cc45ad9ba42928689294c7

After deploying this branch, I tried to re-deploy but the test server failed to even start deploying due to the error

Reply:
@davidsiaw Thank you for catching.
it looks like it doesn't not throw same error when I call this in test with empty name.
I have updated and please take a look when you have time 🙇 
https://github.com/degica/barcelona/pull/686/commits/922c31ae15b1d9bf7467adfcc467cbb99f851455